browser/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  extends: ['mixmax', 'plugin:react/recommended'],
+  parser: 'babel-eslint',
+  settings: {
+    react: {
+      version: '15.5.4'
```

Comment:
What does this do? I foresee it being slightly painful to update in lockstep with React itself (unless we get some sort of warning in CI if it deviates?)

Would it be a bad idea to do (pseudocode) `version: require('./package.json').devDependencies.react`?